pkg/scheduler/framework/v1alpha1/framework_test.go

```diff
@@ -585,6 +589,82 @@ func TestPreFilterPlugins(t *testing.T) {
 
 }
 
+func TestFilterPlugins(t *testing.T) {
```

Comment:
NOTE: these tests only cover a single plugin case. We need cases to cover multiple Filter plugins with different combinations of results, e.g., pl1:Success, pl2:Error, pl1:Error, pl2:Success, etc.

It's also OK if you defer the multiple plugin case. If so, please add a TODO here.

Reply:
I am adding cases to cover multiple plugins cases.